modules/checking-images.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * cli/checking-policy-compliance.adoc
+:_content-type: PROCEDURE
+[id="checking-images_{context}"]
+= Checking images
+
+.Procedure
+
+* Run the following command to check the build-time violations of your security policies in images:
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" image check --image=_<image_name>_
+----
```

Comment:
Let's add the same section regarding the required permissions + force flag here.